content/en/docs/instrumentation/java/springboot.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Spring Boot
+linkTitle: Spring Boot
+aliases:
+  - /docs/java/getting_started
+  - /docs/java/springboot
+  - /docs/instrumentation/java/springboot
+weight: 30
+description: Spring instrumentation for OpenTelemetry Java
+javaInstrumentationVersion: 1.31.0
+cSpell:ignore: springboot autoconfigure 
+---
+
+You can automatically instrument a [Spring Boot](https://spring.io/projects/spring-boot) application using the [OpenTelemetry Java agent](automatic/_index) that uses byte code instrumentation.
+
+The OpenTelemetry starter is another way to instrument your Spring Boot application. It is a [Spring Boot starter](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#using.build-systems.starters).
```

Comment:
```suggestion
The OpenTelemetry Spring Boot Starter is another way to instrument your Spring Boot application. It is a [Spring Boot starter](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#using.build-systems.starters).
```

I am not 100% sure what the exact name of this should be, but Starter should be capitalized and it should be part of the name that this is for Spring